target/linux/ramips/dts/mt7621_buffalo_wsr-2533dhpls.dts

```diff
@@ -0,0 +1,236 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/leds/common.h>
+
+/ {
+	compatible = "buffalo,wsr-2533dhpls", "mediatek,mt7621-soc";
+	model = "Buffalo WSR-2533DHPLS";
+
+	chosen {
+		bootargs = "console=ttyS0,115200n8";
+	};
+
+	aliases {
+		led-boot = &led_power;
+		led-failsafe = &led_diag;
+		led-running = &led_power;
+		led-upgrade = &led_power;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		router_red {
+			label = "red:router";
+			gpios = <&gpio 4 GPIO_ACTIVE_LOW>;
+		};
+		wireless_red {
+			label = "red:wireless";
+			gpios = <&gpio 8 GPIO_ACTIVE_LOW>;
+		};
+		internet_red {
+			label = "red:internet";
+			gpios = <&gpio 10 GPIO_ACTIVE_LOW>;
+		};
+		internet_green {
+			color = <LED_COLOR_ID_GREEN>;
+			label = "green:internet";
+			gpios = <&gpio 12 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "mt7530-0:04:1Gbps";
+		};
+
+		led_power: power_green {
+			function = LED_FUNCTION_POWER;
+			color = <LED_COLOR_ID_GREEN>;
+			label = "green:power";
```

Comment:
`label` seems redundant here

Reply:
Thanks! I didn't know what the function/color does.
This is my first attempt at a contribution and I have a lot to learn.

I have to double check colors again, some "red" might be "amber" instead.